Japanese translation updated

ktyubeshi opened the pull request.

mstrap commented:
Thanks! It seems that line endings have been changed for every line. From my understanding, this is CRLF -> LF what looks good to me (I didn't realize it was CRLF before). Still, can we have a separate commit for the line ending conversion, please?

ktyubeshi commented:
Split commits for translation and EOL changes. Please check it.

I didn't intend to change CRLF -> LF in this commit (tool change mistake), but it's a good opportunity, so I'm in favor of unifying them.

I've used several self-made CAT tools, but until recent versions, they didn't consider line ending, so they were saved with CRLF, which is the Windows default.

Recently, I noticed that the line endings in the files output by mapping and SmartGit were LF, so I corrected the line endings to LF. However, I found that the pull request (https://github.com/syntevo/smartgit-translations/pull/53) at that time made it difficult to view the GitHub history, so I intentionally continued to use CRLF.
